platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/AbstractPKIHandler.java

```diff
@@ -103,7 +103,14 @@ public HandlerResult getNormalizedToken(
     } else {
       String errorMsg = "The certificate used to complete the request has been revoked.";
       LOGGER.info(errorMsg);
-      throw new ServletException(errorMsg);
+      try {
+        httpResponse.sendError(
+            HttpServletResponse.SC_FORBIDDEN, "Your certificate has been revoked.");
+        httpResponse.flushBuffer();
+      } catch (Exception e) {
+        LOGGER.error("Error returning 403 to revoked certificate request.");
```

Comment:
❓  Just to clarify does this read:
"Error returning 403 ... " or "Error- returning 403"? 
Just want to make sure that's clear.

Reply:
Yeah, its an error message that you get when you fail to return an error message, so that you can fix the code while you fix the code. It's kinda confusing so I'll just remove that _403_